scripts/multisig.js

```diff
@@ -0,0 +1,164 @@
+const { program } = require('commander');
+const inquirer = require('inquirer');
+
+const { RECOVERY_METHOD_KINDS, TWO_FACTOR_AUTH_KINDS } = require('../constants');
+const {
+    deleteRecoveryMethod,
+    listRecoveryMethodsByAccountId,
+    updateRecoveryMethod,
+} = require('../db/methods/recovery_method');
+
+function is2faRecoveryMethod(kind) {
+    return Object.values(TWO_FACTOR_AUTH_KINDS).includes(kind);
+}
+
+function isAccountValid(accountId) {
+    if (accountId.length === 64 && !accountId.includes('.')) {
+        return true;
+    }
+
+    if (process.env.NEAR_WALLET_ENV.startsWith('testnet')) {
+        return accountId.endsWith('.testnet');
+    }
+
+    return accountId.endsWith('.near');
+}
+
+function isEmail(detail) {
+    return detail.includes('@');
+}
+
+function isEquivalentPhoneNumber(phone1, phone2) {
+    const nonDigits = /\D/g;
+    return phone1.replace(nonDigits, '') === phone2.replace(nonDigits, '');
+}
+
+function printRecoveryMethods(recoveryMethods) {
+    recoveryMethods
+        .sort(({ kind: kindA }, { kind: kindB }) => {
+            if (kindA > kindB) {
+                return 1;
+            }
+            return -1;
+        })
+        .forEach(({ detail, kind }) => {
+            const is2fa = is2faRecoveryMethod(kind);
+            if (kind === RECOVERY_METHOD_KINDS.PHRASE || kind === RECOVERY_METHOD_KINDS.LEDGER) {
+                console.log(`[${kind}]`);
+            } else {
+                console.log(`${is2fa ? '*' : ''}[${kind}]: ${detail}`);
+            }
+        });
+}
+
+async function lookupRecoveryMethods(accountId) {
+    if (!isAccountValid(accountId)) {
+        console.error(`Invalid account ID ${accountId}`);
+        return;
+    }
+
+    const recoveryMethods = await listRecoveryMethodsByAccountId(accountId)
+        .map(({ detail, kind }) => ({ detail, kind }));
+
+    if (!recoveryMethods.length) {
+        console.log(`\n\nNo recovery methods found for ${accountId}`);
+        return;
+    }
+
+    console.log(`\n\nFound ${recoveryMethods.length} recovery method(s) for ${accountId}:`);
+    printRecoveryMethods(recoveryMethods);
+}
+
+async function transferMultisig({ accountId, current, desired }) {
+    if (!isAccountValid(accountId)) {
+        console.error(`Invalid account ID ${accountId}`);
+        return;
+    }
+
+    const isCurrentEmail = isEmail(current);
+    const isDesiredEmail = isEmail(desired);
+
+    if (!isDesiredEmail) {
+        console.error(`Invalid email: ${desired}. The 2FA method can only be set to email.`);
+        return;
+    }
+
+    const currentMethodKind = isCurrentEmail ? TWO_FACTOR_AUTH_KINDS.EMAIL : TWO_FACTOR_AUTH_KINDS.PHONE;
+    const [sms2faMethod] = await listRecoveryMethodsByAccountId(accountId)
```

Comment:
```suggestion
    const [current2faMethod] = await listRecoveryMethodsByAccountId(accountId)
```
This could also be `email` right?